lib/indexer/tag_lookup.rb

```diff
@@ -27,6 +28,23 @@ def find_document_from_content_api(link)
       end
     end
 
+    def add_self_links(doc_hash)
+      # Consider an organisation page to linked to itself.
+      # This means that when filtering on an organisation,
+      # the organisation page gets included in the search results.
+      #
+      # This deliberately doesn't match up with the canonical representation
+      # of the organisation in the publishing api, since self-linking has
+      # a very fuzzy meaning: ids in links can mean both the thing (HMRC)
+      # and the content representing the thing (the HMRC home page).
+      if doc_hash["format"] == "organisation" && doc_hash["slug"]
+        doc_hash["organisations"] << doc_hash["slug"]
```

Comment:
Yeah, although its order dependent which is a bit awkward. It's initialised in `add_tags_from_artefact`.